src/mod/endpoints/mod_sofia/sofia.c

```diff
@@ -3784,18 +3784,26 @@ static void parse_gateways(sofia_profile_t *profile, switch_xml_t gateways_tag,
 			goto skip;
 		}
 
-		if (gwname && strcmp(gwname, name)) {
+		if (gwname && strcmp(gwname, n
```

Comment:
hmm

Reply:
There is a conflict with upstream master. Please resolve properly.

Reply:
Resolved